recipe/meta.yaml

```diff
@@ -120,13 +120,13 @@ outputs:
         - openjpeg    # [build_platform != target_platform]
         - zlib        # [build_platform != target_platform]
       host:
-        - boost-cpp {{ boost_cpp }}
         - cairo
         - curl
         - fontconfig
         - freetype
         - gettext
         - glib
+        - libboost-devel
```

Comment:
use `libboost-devel` because of the run-dep (now handled through the run-export)...